client/packages/common/src/intl/utils/DateUtils.ts

```diff
@@ -60,6 +60,17 @@ const formatIfValid = (
   }
 ): string => (isValid(date) ? format(date, dateFormat, options) : '');
 
+// Adds the current time to a date object (that presumably has 00:00 as its time
+// component) -- does not mutate input Date object)
```

Comment:
Nit: you can remove the 00:00 comment now...

Reply:
Hmm, I kind of meant in that you would presumably only want to use this method for raw dates (which have local midnight by default), not for dates that already have meaningful time values. It wasn't really meant to be a requirement.